R/RunDiagnostics.R

```diff
@@ -83,6 +99,8 @@ runCohortDiagnostics <- function(packageName = NULL,
                                  runCohortOverlap = TRUE,
                                  runCohortCharacterization = TRUE,
                                  covariateSettings = FeatureExtraction::createDefaultCovariateSettings(),
+                                 runTemporalCohortCharacterization = FALSE,
+                                 temporalCovariateSettings = NULL,
```

Comment:
Why not simply assign the temporal covariate settings you recommend? That way, you don't have to have to elaborate explanation above, because people could see what the default is. So:

```r
 runTemporalCohortCharacterization = FALSE,
 temporalCovariateSettings = FeatureExtraction::createTemporalCovariateSettings(useConditionOccurrence = TRUE,
                                                                                useConditionEraStart = TRUE,
etc.